music21/scale/__init__.py

```diff
@@ -2864,6 +2925,38 @@ def __init__(self, tonic=None):
         # self._abstract.buildNetwork()
 
 
+class JazzMinorScale(DiatonicScale):
+    '''
+    The jazz minor scale.
```

Comment:
Thanks! I would think a short description in words would be nice, something like the ascending form of melodic minor but in both directions.